lrmd/lrmd.c

```diff
@@ -286,13 +290,49 @@ create_lrmd_cmd(xmlNode * msg, crm_client_t * client, lrmd_rsc_t *rsc)
     cmd->versioned_attrs = first_named_child(rsc_xml, XML_TAG_RSC_VER_ATTRS);
 
     if (cmd->versioned_attrs) {
-        GHashTable* versioned_params = unpack_versioned_attributes(cmd->versioned_attrs, rsc, cmd->action);
+        GHashTable* versioned_params =
+            unpack_versioned_attributes(cmd->versioned_attrs, XML_TAG_ATTR_SETS, rsc, cmd->action);
 
         cmd->versioned_attrs = copy_xml(cmd->versioned_attrs);
         g_hash_table_foreach(versioned_params, dup_attr, cmd->params);
         g_hash_table_destroy(versioned_params);
     }
 
+    op_versioned_attrs = first_named_child(rsc_xml, XML_TAG_OP_VER_ATTRS);
+    if (op_versioned_attrs) {
+        GHashTable* versioned_params =
+            unpack_versioned_attributes(op_versioned_attrs, XML_TAG_ATTR_SETS, rsc, cmd->action);
+
+        g_hash_table_iter_init(&iter, versioned_params);
+        while (g_hash_table_iter_next(&iter, (gpointer *) &key, (gpointer *) &value)) {
+            g_hash_table_replace(cmd->params, strdup(key), strdup(value));
+            g_hash_table_replace(cmd->params, crm_meta_name(key), strdup(value));
```

Comment:
Let's add a comment that providing meta names for instance attributes is only for backward compatibility, and will be removed in a future release.